about.html

```diff
@@ -0,0 +1,101 @@
+        <li><p class="nav-desk">Chinese</p></li>
+        <li><p class="nav-desk">Logout</p></li>
+      </ul>
+      <nav class="desk-nav">
+        <div>
+          <a href="./index.html"
+            ><img class="image-desk" src="images/desktop-dogo.png" alt="image"
+          /></a>
+        </div>
+        <div class="container-desk">
+          <ul class="list-desk">
+            <li><a class="anchor" href="./about.html">About</a></li>
+            <li><a class="anchor" href="#">Program</a></li>
+            <li><a class="anchor" href="#">Volunteer</a></li>
+            <li><a class="anchor" href="#">Media</a></li>
+            <li><a class="point" href="#">Donate</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <section class="container">
+      <div class="introduction">
+        <p class="intro-text">"Hello, Caring world!"</p>
+        <h1>DOGO CHARITY HEALTH OUTREACH 2022</h1>
+        <p class="paragraph-title">
+          The health outreach is aimed at providing information about recent
+          trends in the health fields through educative activities and free
+          health services to communities. This year, we hope to impact thousands
+          of lives and improve healthcare awareness.
+        </p>
+        <p class="address">
+          For more enquiries about DOGO Charity Health Outreach 2022!
+        </p>
+        <p class="adress-2">dogo@connectedhomeless.org</p>
+      </div>
+    </section>
+    <section class="about">
+      <h4>DOGO Charity Health Outreach Logo</h4>
+      <div class="guide"></div>
+      <p class="paragraph-title">
+        The DOGO charity health outreach logo was designed in remembrance of
+        6years old fatima who died of cholera in August,2021.
+      </p>
+      <!-- <div class="logo-1"> -->
+      <img class="logo" src="images/dogo.png" alt="dogo-logo" />
+      <!-- </div> -->
+    </section>
+    <section>
+      <h3>DOGO Charity Health Outreach history</h3>
+      <div class="guide"></div>
+      <p class="paragraph-title">
+        Checkout out how we held our last two outreaches in Asia and Africa.
+      </p>
+      <div class="image-class">
+        s
+        <img src="images/2020.png" alt="2020 outreach" class="outreach img" />
+        <img
+          src="images/2021(1).png"
+          alt="2021 outreach"
+          class="outreach img-1"
+        />
+      </div>
+    </section>
+    <section class="partner">
+      <h5>Partner</h5>
+      <div class="guide"></div>
+      <ul>
+        <li>
+          <img src="images/barclays.png" alt="cambridge" class="logo-1" />
+        </li>
+        <li><img src="images/WHO.png" alt="WHO" class="logo-2" /></li>
+        <li><img src="images/Google.png" alt="Google" class="logo-2" /></li>
+      </ul>
+    </section>
+    <footer>
+      <img src="images/DOGO(1).png" alt="dogo-logo" class="dogo-logo" />
+      <p class="paragraph">2022 Dogo Charity. All Rights Reserved</p>
+    </footer>
```

Comment:
- [ ] The footer does not have padding at the bottom to give it some breathing space. Kindly add a padding to the bottom.

![image](https://user-images.githubusercontent.com/43995830/157769419-2112f0bd-21b0-4518-96be-a91de58864c8.png)